### ISSUE TITLE
New narrations mentioned

### ISSUE THREAD

**Issue by ParthivDupakuntla** [facebookresearch/imu2clip#6]
## Type of Issue

Select the type of issue:
- [ ] Bug report (to report a bug)
- [ ] Feature request (to request an additional feature)
- [ ] Tracker (I am just using this as a tracker)
- [ ] Refactor request
- [x] Documentation Ask

## Description

In [dataloader_narration](https://github.com/facebookresearch/imu2clip/blob/b00548fb4bf0923db2a443466b8b36b81fcf72bb/dataset/ego4d/dataloader_narration.py#L16), the data path specified appears unknown. I downloaded from cli and there doesn't exist a ego4d_data_narration_new folder. Can you please give some information regarding the same?

**Comment by andreamad8** [comment]
yes, the narration are the same as the on you download from the CLI.

-- Andrea
